app/controllers/admin/users_controller.rb

```diff
@@ -10,6 +10,18 @@ def index
       @users = User.all
     end
 
+    def toggle_confirmation
+      #This action allow admins to manually toggle confirmation state of another user 
+      if user_params[:to_confirm] == 'true'
+        @user.confirm
+      elsif user_params[:to_confirm] == 'false'
+        params[:user].delete(:to_confirm)
+        params[:user][:confirmed_at] = nil
+        @user.update_attributes(user_params)
```

Comment:
The way you save the change here can be like you had done it the first time.
Set the attribute you want, save the object.